### ISSUE TITLE
No text in notification for Jitsi Click to join button

### ISSUE THREAD

**Issue by ankar84** [RocketChat/Rocket.Chat#17496]
### Description:

Users complying that there is no any desktop or mobile notification when someone starts a Jitsi call in private chat.

### Steps to reproduce:

1. Start Jitsi call in private chat
2. See no notification as other user

### Expected behavior:

Notification for Jitsi call in any chat should be solid, consistent and straightforward on desktop or mobile.
Every chat member should get that notification according to notification preference.

### Actual behavior:

User do not get notification about Jitsi call started

### Server Setup Information:

- Version of Rocket.Chat Server: 3.1.1
- Operating System: CentOS7
- Deployment Method: docker
- Number of Running Instances: 13
- DB Replicaset Oplog: Enabled
- NodeJS Version: 
- MongoDB Version:4.0.17

### Client Setup Information

- Desktop App or Browser Version: Google Chrome, RC.Electron 2.17.7, RC.RN 4.6.0
- Operating System: Windows 10, Android, iOS

**Comment by ankar84** [comment]
It was bug #17689 for me.
Now, when I set mobile notification to All messages (Default) I got push notification for started by other user Jitsi call in DM. 
![Screenshot_2020-05-19-22-53-04-681_com miui home](https://user-images.githubusercontent.com/4023037/82685255-1c21a200-9c7e-11ea-86aa-02a6f612bb2d.jpg)

And in private chat I got push notification, but with delay in 3-5 minutes.
![IMG_20200522_224200](https://user-images.githubusercontent.com/4023037/82685269-22b01980-9c7e-11ea-806c-dcfa821462b6.jpg)
But now problem is that notification is empty. Text about Jitsi conference should be in that notification, I think.
